src/main/java/com/swervedrivespecialties/swervelib/ctre/Falcon500SteerControllerFactoryBuilder.java

```diff
@@ -224,5 +224,13 @@ public double getStateAngle() {
 
             return motorAngleRadians;
         }
+
+        @Override
+        public void resetSteerPosition() {
+            resetIteration = 0;
+            double absoluteAngle = absoluteEncoder.getAbsoluteAngle();
+            motor.setSelectedSensorPosition(absoluteAngle / motorEncoderPositionCoefficient);
+            currentAngleRadians = absoluteAngle;
```

Comment:
currentAngleRadians is never read until it has been modified, so this doesn't seem to do anything. Perhaps making currentAngleRadians itself local may be a decent.

Reply:
This originally was a local variable and was only a field when I was making the reset process its own method. I reverted this change in the NeoSteerControllerFactory but not for Falcon500